paasta_tools/cli/cmds/rollback.py

```diff
@@ -318,4 +317,11 @@ def paasta_rollback(args: argparse.Namespace) -> int:
                 action="rollback", action_details=audit_action_details, service=service
             )
 
+    if returncode == 0:
+        print(
+            PaastaColors.cyan(
+                f"Warning: Don't forget to revert in git as well. Use 'git revert {rolled_back_from.sha}'"
```

Comment:
minor-nit: Use 'git revert {rolled_back_from.sha}_, and go through the normal push process._